.gitignore

```diff
@@ -6,4 +6,5 @@ launchSettings.json
 artifacts/
 msbuild.binlog
 .vscode/
-BenchmarkDotNet.Artifacts/
\ No newline at end of file
+BenchmarkDotNet.Artifacts/
+node_modules/
```

Comment:
If we're adding `node_modules` here, then can we also *remove* it from being repeated in any other `.gitignore` files in subdirectories? Otherwise we can continue gitignoring it only in the subdirectories.

Reply:
Hey @SteveSandersonMS 

Yeah, I can remove from the subdirectories. In fact, there is no need for having multiple .gitignore on subdirectories. The code base is relatively small, so having one on the root should be enough.

The way it is now, in the subdirectory, for whatever reason was still making git to try add _node_modules_ to my changelist.

Do you want me to remove the ones from the subdirectories or just remove the _node_modules_ entry on those?

Thanks

Reply:
Ok. So for now I'll remove it from the .gitignore.

Reply:
Just for the reference:

![image](https://user-images.githubusercontent.com/4714040/39948106-d87ecd56-554a-11e8-91dd-6ab4b09c4328.png)